lib/tasks/marks.rake

```diff
@@ -43,23 +44,27 @@ namespace :db do
         else
           random_mark = rand(0..1)
         end
-        on_result_creation_mark = Mark.find_by(result_id:     result.id,
-                                               markable_id:   criterion.id,
-                                               markable_type: criterion.class.to_s)
-        on_result_creation_mark.update_attribute(:mark, random_mark)
-        result.save
+        marks << Mark.new(result_id:     result.id,
```

Comment:
Please put the `result_id` on a new line (this will help align the arguments to the method)